packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -109,85 +117,81 @@ def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(dcos_api_sess
-    """
-
-    test_uuid = uuid.uuid4().hex
-    octarine_id = uuid.uuid4().hex
-    proxy = ('"http://127.0.0.1:$(/opt/mesosphere/bin/octarine ' +
-             '--client --port {})"'.format(octarine_id))
-    check_command = 'curl --fail --proxy {} marathon.mesos.mydcos.directory'.format(proxy)
-
-    app_definition = {
-        'id': '/integration-test-app-octarine-http-{}'.format(test_uuid),
-        'cpus': 0.1,
-        'mem': 128,
-        'ports': [0],
-        'cmd': '/opt/mesosphere/bin/octarine {}'.format(octarine_id),
-        'disk': 0,
-        'instances': 1,
-        'healthChecks': [{
-            'protocol': 'COMMAND',
-            'command': {
-                'value': check_command
-            },
-            'gracePeriodSeconds': 5,
-            'intervalSeconds': 10,
-            'timeoutSeconds': 10,
-            'maxConsecutiveFailures': 3
-        }]
-    }
-
-    dcos_api_session.marathon.deploy_and_cleanup(app_definition)
-
-
-def test_octarine_srv(dcos_api_session, timeout=30):
-    """
-    Test resolving SRV records through octarine.
-    """
-
-    # Limit string length so we don't go past the max SRV record length
-    test_uuid = uuid.uuid4().hex[:16]
-    octarine_id = uuid.uuid4().hex
-    proxy = ('"http://127.0.0.1:$(/opt/mesosphere/bin/octarine ' +
-             '--client --port {})"'.format(octarine_id))
-    port_name = 'pinger'
-    cmd = ('/opt/mesosphere/bin/octarine {} & '.format(octarine_id) +
-           '/opt/mesosphere/bin/python -m http.server ${PORT0}')
-    raw_app_id = 'integration-test-app-octarine-srv-{}'.format(test_uuid)
-    check_command = 'curl --fail --proxy {} _{}._{}._tcp.marathon.mesos.mydcos.directory'.format(
-        proxy,
-        port_name,
-        raw_app_id)
-
-    app_definition = {
-        'id': '/{}'.format(raw_app_id),
-        'cpus': 0.1,
-        'mem': 128,
-        'cmd': cmd,
-        'disk': 0,
-        'instances': 1,
-        'portDefinitions': [{
-            'port': 0,
-            'protocol': 'tcp',
-            'name': port_name,
-            'labels': {}
-        }],
-        'healthChecks': [{
-            'protocol': 'COMMAND',
-            'command': {
-                'value': check_command
-            },
-            'gracePeriodSeconds': 5,
-            'intervalSeconds': 10,
-            'timeoutSeconds': 10,
-            'maxConsecutiveFailures': 3
-        }]
-    }
-
-    dcos_api_session.marathon.deploy_and_cleanup(app_definition)
+def test_octarine(dcos_api_session, timeout=30):
+    # This app binds to port 80. This is only required by the http (not srv)
+    # transparent mode test. In transparent mode, we use ".mydcos.directory"
+    # to go to localhost, the port attached there is only used to
+    # determine which port to send traffic to on localhost. When it
+    # reaches the proxy, the port is not used, and a request is made
+    # to port 80.
+
+    octarine = "/opt/mesosphere/bin/octarine"
```

Comment:
Why is this a variable at all? Its never mutated and could just be hard coded into the octarine_runner method.